Automattic/wp-calypso
# calypso-config: Extract `calypso/config` into new @automattic/calypso-config package

#### Changes proposed in this Pull Request

* Extracts `calypso/config` into its own `@automattic/calypso-config` package.

#### Testing instructions



* Ensure build is successful
* Ensure unit tests pass
* Ensure you can still enable flags via cookies and the `flags` parameter
* Smoke test the app to make sure configuration is still working

Note: There are various linting errors that I'm ignoring as out of scope for this PR. None of them are new.
